bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,137 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+from typing import Literal
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+TOP_WORDS_FILE_PATH = Path("bot/resources/fun/top_1000_used_words.txt").resolve()
+# defining a list of images that will be used for the game to represent the hangman person
+IMAGES = [
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472355056295946/hangman0.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472756744814613/hangman1.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472808699629578/hangman2.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472862441267230/hangman3.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472950991396864/hangman4.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472999431430204/hangman5.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883473051277226015/hangman6.png",
+]
+
+
+class Hangman(commands.Cog):
+    """
+    Cog for the Hangman game.
+
+    Hangman is a classic game where the user tries to guess a word, with a limited amount of tries.
+    """
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        with TOP_WORDS_FILE_PATH.open(mode="r", encoding="utf-8") as file:
+            self.all_words = [line.strip('\n') for line in file.readlines()]
```

Comment:
Since `TOP_WORDS_FILE_PATH` is only used here, I believe you can just do this at the top of the file instead.

```suggestion
with Path("bot/resources/fun/top_1000_used_words.txt").open("r", encoding="utf-8") as f:
    ALL_WORDS = [line.strip('\n') for line in f.readlines()]
```